main.js

```diff
@@ -2,6 +2,9 @@ const {app, BrowserWindow} = require('electron')
 const path = require('path')
 const url = require('url')
 
+// Let electron reloads by itself when webpack watches changes in ./app/
+require('electron-reload')(__dirname)
```

Comment:
@mohamed-moanis  I'm just wondering how this would behave given that this is a dev-dependency and we would ignore dev-dependencies during the production build.